tests/unit/test_xtcedef.py

```diff
@@ -1523,16 +1524,73 @@ def test_float_parameter_type(xml_string: str, expectation):
          xtcedef.Packet(0b1111111111010110.to_bytes(length=2, byteorder='big'),
                             parsed_data={'PKT_APID': parser.ParsedDataItem('PKT_APID', 1101)}),
          -82.600000),
+        # Test MIL 1750A encoded floats.
+        # Test values taken from: https://www.xgc-tek.com/manuals/mil-std-1750a/c191.html#AEN324
+        (xtcedef.FloatParameterType(
+            'MIL_1750A_FLOAT',
+            xtcedef.FloatDataEncoding(32, encoding="MIL-1750A")),
+         xtcedef.Packet(b'\x7f\xff\xff\x7f'),
+         0.9999998 * (2 ** 127)),
+        (xtcedef.FloatParameterType(
+            'MIL_1750A_FLOAT',
+            xtcedef.FloatDataEncoding(32, encoding="MIL-1750A")),
+         xtcedef.Packet(b'\x40\x00\x00\x7f'),
+         0.5 * (2 ** 127)),
+        (xtcedef.FloatParameterType(
+            'MIL_1750A_FLOAT',
+            xtcedef.FloatDataEncoding(32, encoding="MIL-1750A")),
+         xtcedef.Packet(b'\x50\x00\x00\x04'),
+         0.625 * (2 ** 4)),
+        (xtcedef.FloatParameterType(
+            'MIL_1750A_FLOAT',
+            xtcedef.FloatDataEncoding(32, encoding="MIL-1750A")),
+         xtcedef.Packet(b'\x40\x00\x00\x01'),
+         0.5 * (2 ** 1)),
+        (xtcedef.FloatParameterType(
+            'MIL_1750A_FLOAT',
+            xtcedef.FloatDataEncoding(32, encoding="MIL-1750A")),
+         xtcedef.Packet(b'\x40\x00\x00\x00'),
+         0.5 * (2 ** 0)),
+        (xtcedef.FloatParameterType(
+            'MIL_1750A_FLOAT',
+            xtcedef.FloatDataEncoding(32, encoding="MIL-1750A")),
+         xtcedef.Packet(b'\x40\x00\x00\xff'),
+         0.5 * (2 ** -1)),
+        (xtcedef.FloatParameterType(
+            'MIL_1750A_FLOAT',
+            xtcedef.FloatDataEncoding(32, encoding="MIL-1750A")),
+         xtcedef.Packet(b'\x40\x00\x00\x80'),
+         0.5 * (2 ** -128)),
+        (xtcedef.FloatParameterType(
+            'MIL_1750A_FLOAT',
+            xtcedef.FloatDataEncoding(32, encoding="MIL-1750A")),
+         xtcedef.Packet(b'\x00\x00\x00\x00'),
+         0.0 * (2 ** 0)),
+        (xtcedef.FloatParameterType(
+            'MIL_1750A_FLOAT',
+            xtcedef.FloatDataEncoding(32, encoding="MIL-1750A")),
+         xtcedef.Packet(b'\x80\x00\x00\x00'),
+         -1.0 * (2 ** 0)),
+        (xtcedef.FloatParameterType(
+            'MIL_1750A_FLOAT',
+            xtcedef.FloatDataEncoding(32, encoding="MIL-1750A")),
+         xtcedef.Packet(b'\xBF\xFF\xFF\x80'),
+         -0.5000001 * (2 ** -128)),
+        (xtcedef.FloatParameterType(
+            'MIL_1750A_FLOAT',
+            xtcedef.FloatDataEncoding(32, encoding="MIL-1750A")),
+         xtcedef.Packet(b'\x9F\xFF\xFF\x04'),
+         -0.7500001 * (2 ** 4)),
     ]
 )
 def test_float_parameter_parsing(parameter_type, packet, expected):
     """Test parsing float parameters"""
     raw, derived = parameter_type.parse_value(packet)
+    # NOTE: These results are compared with a relative tolerance due to the imprecise storage of floats
```

Comment:
You can avoid the numpy import and use `pytest.approx`: https://docs.pytest.org/en/latest/reference/reference.html#pytest.approx

```python
    expected = pytest.approx(expected, rel=0.000001)
    if derived:
        assert derived == expected
    else:
        assert raw == expected
```

Reply:
Oh nice! I didn't know about this. Will do.